src/ESPEasy_buildinfo.h

```diff
@@ -20,7 +20,7 @@
 #endif // if defined(ESP32)
 
 
-#define BUILD                           20104    // git version e.g. "20103" can be read as "2.1.03" (stored in int16_t)
+#define BUILD                           20105    // git version e.g. "20103" can be read as "2.1.03" (stored in int16_t)
```

Comment:
Are you sure it isn't already set to 20105?
Are you up-to-date with the current HEAD of mega branch?

Reply:
Yes, i'm 100% sure.
Maybe you haven't pushed some changes?

Reply:
Just given the fact we have this merge conflict, I am sure you're not basing your PR on the latest HEAD of mega:
```
<<<<<<< feature/ethernet_support
#ifdef HAS_ETHERNET
  const unsigned int SettingsStructSize = (252 + 82 * TASKS_MAX);
#else
  const unsigned int SettingsStructSize = (244 + 82 * TASKS_MAX);
#endif
=======
  const unsigned int SettingsStructSize = (248 + 82 * TASKS_MAX);
>>>>>>> mega
```

Reply:
I just resolved this merge conflict, so you should now try to fetch the updated PR.
N.B. there is also some code referring the 20105 build (as `< 20105` ) to initialize newly added members to the settings struct.

Reply:
Damn, i forget i was fetching origin not upstream..Sorry, and thanks for help :)

Reply:
I've rebased my branch, let's keep linear history without merges...

Reply:
The reason I did fix the merge conflict was because there have been additions to the SettingsStruct, so that's one thing we really should make sure it won't get lost in merges.